File: VictimsHelp/VictimsHelp/VictimsHelp.DAL/Entities/Message.cs
using System;

namespace VictimsHelp.DAL.Entities
{
    public class Message : BaseEntity
    {
        public string SenderEmail { get; set; }
        public string ReceiverEmail { get; set; }
        public string SenderName { get; set; }
        public string Text { get; set; }
        public DateTime DateTime { get; set; }

        public User Sender { get; set; }
        public User Receiver { get; set; }
    }
}

File: MobileApp/victims_help/lib/models/message.dart
class Message{
  String id;
  String senderEmail;
  String receiverEmail;
  String senderName;
  String text;
  String dateTime;

  Message({this.id, this.senderEmail, this.receiverEmail, this.senderName, this.text, this.dateTime});

  factory Message.fromJson(Map<String, dynamic> responseData) {
    return Message(
      id: responseData['id'],
      senderEmail: responseData['senderEmail'],
      receiverEmail: responseData['receiverEmail'],
      senderName: responseData['senderName'],
      text: responseData['text'],
      dateTime: responseData['dateTime']
    );
  }
}